core/src/main/java/com/linecorp/armeria/client/HttpClientDelegate.java

```diff
@@ -207,6 +207,8 @@ void invoke0(Channel channel, ClientRequestContext ctx,
                 return;
             }
 
+            channel.pipeline().addFirst(new HttpClientFirstTransferLogger(ctx));
```

Comment:
I think this does not work as intended because:

- HTTP/2 is multiplex protocol. There's no way to tell the byte you just received is yours.
- We will have as many `*FirstTransferLogger`s as the number of pending requests, i.e. very deep call stack and poor performance.